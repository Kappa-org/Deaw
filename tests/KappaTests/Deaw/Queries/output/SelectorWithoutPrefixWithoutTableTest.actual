Kappa\Deaw\Queries\Selector::__set_state(array(
	'selects' => array('foo' => '', 'bar' => 'bar2'),
	'tableName' => NULL,
	'prefix' => NULL,
))