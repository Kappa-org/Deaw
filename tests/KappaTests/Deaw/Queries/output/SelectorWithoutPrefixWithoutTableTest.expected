foo,bar as bar2